Mathlib/NumberTheory/NumberField/ExistsRamified.lean

```diff
@@ -0,0 +1,59 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.NumberTheory.NumberField.Discriminant.Basic
+import Mathlib.NumberTheory.NumberField.Discriminant.Different
+import Mathlib.NumberTheory.RamificationInertia.Galois
+
+/-!
+# Every number field has a ramified prime over `ℚ`
+...except `ℚ` itself.
```

Comment:
I'm wondering... is the contrapositive useful as well?

A number field that is unramified over Q has finrank = 1.